app/models/participant_profile/ecf.rb

```diff
@@ -83,7 +92,7 @@ def self.archivable(for_cohort_start_year:, restrict_to_participant_ids: [])
     query = left_joins(:participant_declarations, schedule: :cohort)
       # Exclude participants that have any induction records that are not FIP.
       .where.not(id: not_fip_induction_records.select(:participant_profile_id))
```

Comment:
The requirements for the archive logic are at the bottom of [Nathan's word doc](https://educationgovuk.sharepoint.com.mcas.ms/:w:/s/TeacherServices/EYgnylIh7ctAtknjsx6X9rIBz-P3i9isE4anE6GnWhl5Lw?e=EpC3uQ), he's also included this explanation for those undeclared participants that will not be archived:

> This means that there will be some undeclared participants who are not archived. A tiny number of these participants may require FIP/CIP training in the 2024 cohort (estimate 10s not 100s). It may be preferable for these participants to be transferred manually by DfE admin users if flagged by providers/helpdesk.

Reply:
Ok, if he wants to transferred them manually when flagged by providers/helpdesk, I am happy to leave the code as it is now.